packages/firebase_messaging/firebase_messaging/example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.0.1'
+        classpath 'com.android.tools.build:gradle:7.4.1'
```

Comment:
I think we can keep gradle 8.0.1, it shouldn't cause issues

Reply:
I'm afraid it does cause issues, it requires Java 17

Reply:
But this is the gradle file of the example app, should not impact the release

Reply:
True, it won't impact the release, but anyone trying to build the example app won't be able to without java 17. It's an easy win.